unsupported data type Int64 for function ceil

**Describe the bug**
The ceil function passes a bigint type field, and the task reports an error message of unsupported data type Int64 for function ceil
**To Reproduce**
`
-- 报错sql
SELECT
  file_size,  -- bigInt类型
  ceil(file_size),  -- 报错语法处
  ceil(100.1),
  ceil(200 / 100.1),
  ceil(file_size / 100.1)
from
  ads.hdfs_meta_raw_data_wsk_test02;

-- 错误栈
Job aborted due to stage failure: Task 0 in stage 3.0 failed 4 times, most recent failure: Lost task 0.3 in stage 3.0 (TID 5) (hz-hadoop-dd-98-5-24 executor 1): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to hdfs://nameHAservice/user/hive/warehouse/ads.db/ads_gov_storage_summary.
	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:788)
	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:891)
	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:891)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:92)
	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:161)
	at org.apache.spark.scheduler.Task.run(Task.scala:139)
	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$3(Executor.scala:554)
	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:1530)
	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:557)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: task panics: Execution error: Execution error: output_with_sender[Project] 
error: Execution error: output_with_sender[Project]: output() returns error: Execution error: Unsupported data type Int64 for function ceil`

**Expected behavior**
Looking forward to the job running smoothly without any errors

--- a/spark-extension/src/main/scala/org/apache/spark/sql/blaze/NativeConverters.scala
+++ b/spark-extension/src/main/scala/org/apache/spark/sql/blaze/NativeConverters.scala
@@ -843,13 +843,17 @@ object NativeConverters extends Logging {
           }
         }
       case e: Ceil if !e.dataType.isInstanceOf[DecimalType] =>
-        buildExprNode {
-          _.setTryCast(
-            pb.PhysicalTryCastNode
-              .newBuilder()
-              .setExpr(buildScalarFunction(pb.ScalarFunction.Ceil, e.children, e.dataType))
-              .setArrowType(convertDataType(e.dataType))
-              .build())
+        if (e.child.dataType.isInstanceOf[LongType]) {
+          convertExprWithFallback(e.child, isPruningExpr, fallback)
+        } else {
+          buildExprNode {
+            _.setTryCast(
+              pb.PhysicalTryCastNode
+                .newBuilder()
+                .setExpr(buildScalarFunction(pb.ScalarFunction.Ceil, e.children, e.dataType))
+                .setArrowType(convertDataType(e.dataType))
+                .build())
+          }
         }
 
       // TODO: datafusion's round() has different behavior from spark

